src/main/java/org/powerflows/dmn/io/yaml/YamlReader.java

```diff
@@ -30,7 +30,7 @@
 
 public class YamlReader extends AbstractDecisionReader {
 
-    private final Yaml yaml = new Yaml(new YamlDecisionConstructor());
+    private final Yaml yaml = new Yaml(new PowerflowsYamlConstructor());
```

Comment:
Powerflows in the name - too verbose imho

Reply:
yeah, simplified to same convention as other snake yaml overrides.